src/elm/View/MainText.elm

```diff
@@ -19,8 +19,16 @@ viewTop sectionId mainTextList =
     in
     case maybeMainTextList of
         Just mainTextHead ->
-            Html.div [ Html.Attributes.class "main-texts" ]
-                [ mainTextHead ]
+            case sectionId of
```

Comment:
Would this be better as an `If ... else` or do we predict nuance in other sections?